src/Control/PathfinderPageController.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace CodeCraft\Pathfinder\Control;
+
+use CodeCraft\Pathfinder\Extension\PathfinderControllerExtension;
+use CodeCraft\Pathfinder\Model\PathfinderPage;
+use PageController;
+
+/**
+ * The controller for {@see PathfinderPage}
+ */
+class PathfinderPageController extends PageController
+{
+
+    /**
+     * @var string
+     */
+    private static $extensions = [
+        PathfinderControllerExtension::class,
```

Comment:
```suggestion
        PathfinderPageControllerExtension::class,
```

Should this match the class name/page name?

Reply:
Nah, because it's not explicitly a `PageController` extension, it could extend something like `ElementController`, so I named it for the shared parent `Controller` class